source/_integrations/stream.markdown

```diff
@@ -24,6 +24,19 @@ The `stream` integration is automatically loaded by `default_config` and enabled
 stream:
 ```
 
+## LL-HLS
+
+Beginning with Home Assistant version 2021.10, there is the option to enable LL-HLS for lower latency. Please note that LL-HLS has strict timing and network requirements that can be hard to meet, so this option may not work on all setups. As HTTP/2 is required by the LL-HLS specification, use of an HTTP/2 enabled reverse proxy is strongly recommended.
```

Comment:
Small change, as we don't keep a changelog in our docs. The docs always match the latest version of Home Assistant.


```suggestion
There is the option to enable LL-HLS for lower latency. Please note that LL-HLS has strict timing and network requirements that can be hard to meet, so this option may not work on all setups. As HTTP/2 is required by the LL-HLS specification, the use of an HTTP/2 enabled reverse proxy is strongly recommended.
```